pallets/liquidity-pools/src/weights_generated.rs

```diff
@@ -0,0 +1,78 @@
+
+//! Autogenerated weights for `pallet_liquidity_pools`
```

Comment:
NOTE: Committed solely for demonstration purpose, will be removed before merging!